### ISSUE TITLE
Admin Passwort für E-Mail Postfächer

### ISSUE THREAD

**Issue by vogelor** [mittwald/feature-requests#147]
**Welches Problem möchtest du lösen? Wann tritt es auf?**
Die Kunden rufen immer wieder an und haben Probleme mit Ihrem E-Mail Postfach. Ich würde denen gerne helfen, habe aber KEINEN Zugriff auf deren E-Mail Postfächer.

**Welche Lösungsideen hast du?**
Es sollte einen Knopf "Admin-Login erstellen" geben. Dieser erstellt ein WEITERES Passwort für das Postfach (es gibt dann also "kurz" 2) und teilt mit das PWD mit. Ich kann mich dann am Postfach anmelden und dem Kunden helfen.
NACHTS um 3:00 Uhr werden sämtliche Admin -PWD wieder gelöscht.

**Comment by twiesing** [comment]
Alternativ könnte ich mir auch ein Single Sign On ins Postfach vorstellen (über Roundcube), welchen der Endkunde zuvor freischalten muss bzw. für jeden neuen Login temporär freigeben kann.

**Comment by vogelor** [comment]
> Alternativ könnte ich mir auch ein Single Sign On ins Postfach vorstellen (über Roundcube), welchen der Endkunde zuvor freischalten muss bzw. für jeden neuen Login temporär freigeben kann.

Das sehe ich nicht als Alternative. Wenn jemand ein neues E-Mail Postfach "bekommt", dann wird das Postfach eingerichtet. Danach wird einmalig Outlook (oder Thunderbird) eingerichtet. Keiner meiner Kunden weiß sein Passwort für das Postfach. Deshalb kann er sich auch nicht selbst am Webmailer anmelden.

**Comment by lmrkavogt** [comment]
Man könnte es auch via SSO lösen und dem Inhaber des Postfachs ggf zeitversetzt eine Benachrichtigung per Mail über den Admin-Login zukommen lassen.

SSO finde ich eine gute Sache und wahrscheinlich auch einfacher als ein zusätzliches Admin-Kennwort.

**Comment by vogelor** [comment]
Also ich habe jetzt seit 34 Jahren ein Systemhaus. Seit 1997 richte ich Internet-Seiten und damit auch E-Mail Postfächer ein. 

Ich kenne wirklich kaum einen Kunden, der sein Postfach-PWD kennt. Nur ein paar wenige, die wissen, WO sie es AUFGESCHRIEBEN haben. 

Wenn also SSO, dann ist die Frage, WOMIT? Mit dem E-Mail Postfach scheidet aus, denn dass kennen die Kunden nicht (und ich ja auch nicht). Das müsste dann ja irgendwie MEIN SSO sein. Das macht in meinen Augen keinen Sinn. ICH muss ja ins Postfach, nicht der Kunde...

**Comment by lmrkavogt** [comment]
Naja, es geht ja darum, dass du als Admin in das Postfach sollst/musst.

Das machst du folglich von der Konsole = mStudio aus und drückst hier auf den Button "Login" und verbindest dich per SSO in das Postfach.

Ob der Kunde sein pwd kennt oder nicht ist in dem Moment dann egal.

**Comment by vogelor** [comment]
O.K. Dann habe ich es falsch verstanden. Ich hatte verstanden, der KUNDE soll sich per SSO am Postfach anmelden und es mir dann "freigeben". Wenn ICH da ran komme ist das besser.

**Comment by twiesing** [comment]
Genau, was @lmrkavogt sagt 👍🏻 Kannst du @vogelor noch einmal erläutern, mit welchen Fragen die Kunden auf dich zukommen? Mail-Client-Einrichtung wird es bei der gewünschten Lösung ja vermutlich nicht sein, oder?

**Comment by vogelor** [comment]
Nein, es sind mehr so allgemeine Probleme:
* Ich finde eine Mail nicht (liegt dann im SPAM)
* Mein E-Mail Postfach ist voll, aber das kann nicht sein, ich sehe gar keine Mails (sind in Unterordner oder im Papierkorb)
* Ich bekomme gar keine Mails, ich glaube meine Exchange E-Mail Abholung ist abgestürzt (nein, sie haben gar keine Mails heute ins Postfach bekommen)
* Ein Kunde versucht mir was zu senden, aber ich bekomme nichts - warum geht das nicht (ich sehe die Mail, sie kam um 10:13 und 15 Sekunden an)
Also mehr so Dinge, bei denen der Kunde uns beschuldigt, in Wirklichkeit macht er selbst das aber falsch...

**Comment by vogelor** [comment]
Ein Klassiker ist auch:
"ich kann keine Mails versenden, da stimmt was nicht"
Wenn ich mich dann im Postfach anmelde (Webmailer) und mir selbst eine Test-Mail sende, dann geht es.
Dann weiß ich schon, es liegt am PC des Kunden.

**Comment by twiesing** [comment]
Perfekt, das hilft 💪🏻
